app/views/inbox/_entry.haml

```diff
@@ -22,15 +22,18 @@
             .dropdown-menu.dropdown-menu-right{ role: :menu }
               - if i.question.user_id != current_user.id
                 %a.dropdown-item{ name: "ib-report", data: { q_id: i.question.id } }
-                  %i.fa.fa-warning
+                  %i.fa.fa-fw.fa-warning
                   = t("voc.report")
-              - if i.question.author_is_anonymous && i.question.author_identifier.present?
+              - if current_user.has_role?(:moderator) && i.question.anonymous? && !i.question.generated?
```

Comment:
use the [`User#mod?`](https://github.com/Retrospring/retrospring/blob/41765159a9834ce184050949dd90730bee030db9/app/models/user.rb#L140) method instead of checking for the moderator role on its own